modules/yaml/yaml.go

```diff
@@ -42,6 +42,13 @@ func renderHorizontalHtmlTable(m yaml.MapSlice) string {
 
 		if value != nil && reflect.TypeOf(value).String() == "yaml.MapSlice" {
 			value = renderHorizontalHtmlTable(value.(yaml.MapSlice))
+		} else if value != nil && reflect.TypeOf(value).String() == "[]interface {}" {
```

Comment:
Could we use a [type switch](https://golang.org/doc/effective_go.html#type_switch) instead of manually using the reflection library for the checks on lines 43 and 45?